src/pages/tools/MonsterSlayerPage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import {
+    TypedUseSelectorHook,
+    useDispatch,
+    useSelector as useReduxSelector
+} from 'react-redux';
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+  } from '@material-ui/core';
+
+import { loadStageScores } from '../../store/stageScoreActions';
+import { StageScore } from '../../domain/stageScore';
+import { AppState } from '../../store/rootReducer';
+
+export const useSelector: TypedUseSelectorHook<AppState> = useReduxSelector;
+
+export const MonsterSlayerPage: React.FC<{}> = () => {
+    const stageScores: StageScore[] = useSelector(
+        state => state.stageScoreState.stageScores
+    );
+
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(loadStageScores());
+    }, [dispatch]);
+
+    if (stageScores.length == 0) {
```

Comment:
Hmm I would prefer to stick to a single return option for the React components just for the normal process. Instead of using the if statement like this try using a trinary operator to reduce it to an inline operation so we either get the no active event information or we get the full table.

You can use this segment I used as a reference for login:
https://github.com/Mandalorian007/questland-handbook/blob/master/src/pages/profile/AccountPage.tsx#L87